RestaurantsNearMe/Presentation/Components/Toast.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Toast.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2023-11-25.
+//
+
+import SwiftUI
+
+struct Toast: View {
+  let message: String
+  @Environment(\.deviceOrientation) var deviceOrientation
+
+  var body: some View {
+    RoundedRectangle(cornerRadius: 10.0)
+      .strokeBorder()
+      .fill(Color.clear)
+      .containerRelativeFrame(.horizontal) { layout, _ in
+        layout * UIConstants.toastWidthRatio
+      }
+      .containerRelativeFrame(.vertical) { layout, _ in
```

Comment:
I've never really seen this API before, you could always use a [GeometryReader](https://developer.apple.com/documentation/swiftui/geometryreader) as an alternative. Certain iOS devices break the "portrait" and "landscape" checks - iPads can do split screen for example.